tests/nested/manual/minimal-smoke/task.yaml

```diff
@@ -0,0 +1,41 @@
+summary: execute smoke tests in a nested Ubuntu Core VM that meets the minimal requirements
+
+systems: [ubuntu-16.04-64, ubuntu-18.04-64, ubuntu-20.04-64]
+
+prepare: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+    create_nested_core_vm
+
+execute: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+
+    # see https://docs.ubuntu.com/core/en/#advantages-for-iot for minimum
+    # requirements
+    if is_core_20_nested_system ; then
```

Comment:
perhaps we could have a function in nested.sh to get the nested system that then could be used on other tests as well

Reply:
Played a bit with this, but in the end I still had to have almost identical case block in the test to pick the memory setting. Perhaps when there's another test where a similar pattern appears we can apply a refactor and extract the common part into a helper.